styles.css

```diff
@@ -0,0 +1,92 @@
+* {
+  margin: 0;
+}
+
+.container {
+  width: 375px;
+  height: 718px;
+  background-color: #ebebff;
+}
+
+.main-content {
+  width: 375px;
+  height: 718px;
+  background-color: #fff;
+  border-radius: 1px 1px 1px 40px;
+  background-image: url("./images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] Please kindly make sure your design sticks as much as possible to the Figma design, the background image has a border-radius at the left, please kindly fix it 🙃 😉 

Your design

<img width="284" alt="Screen Shot 2022-09-27 at 17 19 59" src="https://user-images.githubusercontent.com/74501523/192567342-0fac20c5-724d-4e7c-88aa-e2e26895ffe5.png">

Figma design

<img width="265" alt="Screen Shot 2022-09-27 at 17 20 39" src="https://user-images.githubusercontent.com/74501523/192567415-7185ea83-74bf-4a05-b9d5-29cbbbde86cc.png">